testsuite/performance/README.provisioning-parameters.md

```diff
@@ -29,35 +29,18 @@
 |-------------|-------------------------------|-------------------------|-----------------------------------------------------------------------------------------|
 | JVM         | Memory settings               | `infinispan.jvm.memory` | -Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -XX:+DisableExplicitGC |
 
-## CPUs
-
-At the moment it is not possible to dynamically parametrize the number of CPUs for a service via Maven properties or environment variables.
-
-To change the default value (`cpus: 1`) it is necessary to edit the Docker Compose file.
-
-
-### Example: Keycloak service using 2 CPU cores
-
-`docker-compose.yml` and `docker-compose-cluster.yml`:
-```
-services:
-    ...
-    keycloak:
-        ...
-        cpus: 2
-        ...
-```
-
-`docker-compose-crossdc.yml`:
-```
-services:
-    ...
-    keycloak_dc1:
-        ...
-        cpus: 2
-        ...
-    keycloak_dc2:
-        ...
-        cpus: 2
-        ...
-```
+## Docker settings
+
+By default, there are 4 CPU cores allocated: core 0 for monitoring, core 1 for database (MariaDB), and cores 2 and 3 for Keycloak server.
```

Comment:
Will this work correctly when the scaling services up? For example if there are 2 or more instances of Keycloak server, will they all share cores 2-3?

Reply:
Yes, they will all share cores 2-3. This limitation comes from docker-compose way of scaling and there is no immediately available way to set it per started container - AFAIK, one would just need to declare and start N containers with specific cpusets instead of starting one N times.

Reply:
We need to add some notes about how to set  the `cpuset` parameter together with cluster scaling. The value of the `cpuset` (range of cpu cores) depends on the size of the provisioned cluster.

Reply:
Added note for easier reference of what these parameters mean and how they map to docker-compose file values, but let me stress that this document is no performance-tuning guide. The knowledge of how to tune these parameters must come from understanding the impact of the parameter values that among other depends on underlying architecture - which is way beyond the scope of this description of parameters.

Reply:
Yes, the details are out of scope. I just wanted the users to be aware that for scalability testing they need to scale the cpuset together with the size of the Keycloak cluster. This is not apparent from reading the "main" README so when people will be scaling up and forget to increase the cpuset range accordingly they will get bad results.